app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -1,436 +1,164 @@
+      :find_key => [:section, :name]
+    },
+    :container_quota               => {
+      :links    => :container_project,
+      :children => :container_quota_items,
+    },
+    :container_quota_item          => {
+      :find_key => :resource
+    },
+    :container_limit               => {
+      :links    => :container_project,
+      :children => :container_limit_items
+    },
+    :container_limit_item          => {
+      :find_key => [:resource, :item_type]
+    },
+    :container_node                => {
+      :children => [:labels, :computer_system, :container_conditions]
+    },
+    :computer_system               => {
+      :children => [:hardware, :operating_system]
+    },
+    :container_condition           => {
+      :find_key => :name
+    },
+    :container_build               => {
+      :links    => :container_project,
+      :children => :labels
+    },
+    :container_build_pod           => {
+      :links    => :container_build,
+      :children => :labels
+    },
+    :container_image_registry      => {
+      :find_key => [:host, :port]
+    },
+    :container_image               => {
+      :find_key => [:image_ref, :container_image_registry_id],
+      :links    => :container_image_registry
+    },
+    :container_replicator          => {
+      :links    => [:container_project],
+      :children => [:labels, :selector_parts]
+    },
+    :container_group               => {
+      :children => [:container_definitions, :containers, :labels, :node_selector_parts, :container_conditions,
+                    :container_volumes],
+      :links    => [:container_node, :container_replicator, :container_project, :container_build_pod]
+    },
+    :container_definition          => {
+      :children => [:container_port_configs, :container_env_vars, :security_context, :container]
+    },
+    :container_port_config         => {
+
+    },
+    :container_env_var             => {
+      :find_key => [:name, :value, :field_path]
+    },
+    :security_context              => {},
+    :container                     => {
+      :links    => :container_image
+    },
+    :node_selector_part            => {
+      :find_key => [:section, :name]
+    },
+    :container_volume              => {
+      :find_key => :name
+    },
+    :container_service             => {
+      :links    => [:container_groups, :container_project],
+      :children => [:labels, :selector_parts, :container_service_port_configs]
+    },
+    :selector_part                 => {
+      :find_key => [:section, :name]
+    },
+    :container_service_port_config => {},
+    :container_route               => {
+      :links    => [:container_project, :container_service],
+      :children => :labels
+    },
+    :persistent_volume             => {},
+    :container_component_status    => {
+      :find_key => :name
+    },
+  }
+  LINKS = {
+    :container_project        => ->(h, _) { h[:container_project_id] = h.fetch_path(:container_project, :id) },
```

Comment:
@moolitayer why do you send the parent in line 149/159 and here you ignore it in all cases. what is this use case of this parent?